environments/sso_vars.yaml

```diff
@@ -0,0 +1,30 @@
+# Parameters for single-signon.yaml and enable-federation-openidc.yaml
+parameter_defaults:
+  KeystoneTrustedDashboards: https://esi.massopen.cloud/
+  # Copy the IdP name from moc openstack. Should we use a different provider for ESI?
+  KeystoneOpenIdcIdpName: moc
+  # Copy the ProviderMetadataUrl from moc openstack. Same as above.
+  KeystoneOpenIdcProviderMetadataUrl: https://sso.massopen.cloud/auth/realms/moc/.well-known/openid-configuration
+  # I guess OpenID client ID should be the identifier for ESI deployment. So we need a new client ID for it.
+  KeystoneOpenIdcClientId: esi
+  # Copy KeystoneIdentityProviders parameters from moc openstack
+  KeystoneIdentityProviders:
+    moc:
+      remote_id: https://sso.massopen.cloud/auth/realms/moc
+      rules:
+        - local:
+            - user: {name: '{0}'}
+          remote:
+            - {type: OIDC-preferred_username}
+      protocol: openid
+  # The client secret to use when handshaking with your OpenID Connect provider.
+  # single-signon.yaml doesn't have this parameter, where does moc openstack store the secret value?
+  KeystoneOpenIdcClientSecret: <myclientsecret>
+  # Passphrase to use when encrypting data for OpenID Connect handshake.
+  # Same question as above
+  KeystoneOpenIdcCryptoPassphrase: <openstack>
+  # OAuth 2.0 introspection endpoint for mod_auth_openidc. Maybe we can use the same url as MOC openstack?
+  KeystoneOpenIdcIntrospectionEndpoint: <https://myidp.example.test/auth/realms/openstack/protocol/openid-connect/token/introspect>
```

Comment:
This should be `https://sso.massopen.cloud/auth/realms/moc/protocol/openid-connect/token/introspect`.